packages/events-example/README.md

```diff
@@ -1,59 +1,115 @@
+using [Pulumi](https://app.pulumi.com/) to deploy objects into AWS.
+## Prerequisites ##
+
+This example assumes you have:
+
+* [An AWS account](https://aws.amazon.com/free/)
+* [A Pulumi account](https://app.pulumi.com/signup) that's 
+  [connected to your AWS account](https://www.pulumi.com/docs/get-started/aws/begin/)
+* A Nile developer account (see [Appendix](#appendix) for help)
+
+## Configure Your Control Plane ##
+
+1. [Login to the Nile Admin Dashboard](https://nad.thenile.dev/) If you don't 
+  already have one, create a workspace.
+2. Create an entity type named "SkyNet". An simple schema is sufficient for 
+  this example:
+
+```json
+{
+  "name": "SkyNet",
+  "schema": {
+    "type": "object",
+    "properties": {
+      "greeting": {
+        "type": "string"
+      }
+    }
+  }
+}
 ```
-<!-- usagestop -->
-# Commands
-<!-- commands -->
-* [`nile reconcile`](#nile-reconcile)
 
-## `nile reconcile`
+3. Create an organization in your workspace. Name it whatever you'd like.
+4. Create an SkyNet instance in your organization, with a simple greeting:
 
-reconcile nile/pulumi deploys
+```json
+{
+  "greeting": "Come with me if you want to live."
+}
+```
+
+## Configure Your Data Plane ##
+
+You can follow the steps to [create a Pulumi project](https://www.pulumi.com/docs/get-started/aws/create-project/)
+and [deploy the stack](https://www.pulumi.com/docs/get-started/aws/deploy-stack/).
+
+More concisely, in a separate directory:
 
+```bash
+$ mkdir nile-examples && cd nile-examples
+$ pulumi new aws-typescript
 ```
-USAGE
-  $ nile reconcile [--json] [--basePath <value>] [--workspace <value>] [--email <value>] [--password <value>]
-    [--organization <value>] [--entity <value>] [-s] [--region <value>]
-
-FLAGS
-  -s, --status            check current status of your control and data planes
-  --basePath=<value>      [default: http://localhost:8080] root URL for the Nile API
-  --email=<value>         [default: developer@demo.com] developer email address
-  --entity=<value>        an entity type in your Nile workspace
-  --organization=<value>  an organization in your Nile workspace
-  --password=<value>      [default: very_secret] developer password
-  --region=<value>        [default: us-west-2] AWS region
-  --workspace=<value>     [default: dev] your Nile workspace name
-
-GLOBAL FLAGS
-  --json  Format output as json.
-
-DESCRIPTION
-  reconcile nile/pulumi deploys
+
+and accept the defaults:
+
 ```
-<!-- commandsstop -->
+project name: (nile-examples) 
+project description: (A minimal AWS TypeScript Pulumi program) 
+Created project 'nile-examples'
+
+Please enter your desired stack name.
+To create a stack in an organization, use the format <org-name>/<stack-name> (e.g. `acmecorp/dev`).
+stack name: (dev) 
+Created stack 'dev'
+
+aws:region: The AWS region to deploy into: (us-east-1)
+Saved config
+```
+
+Run `pulumi up` to ensure that you've configured Pulumi correctly. This will
```

Comment:
Is there a requirement to have the pulumi CLI installed?

Reply:
yeah, i'll note that in the docs as part of cleaning up what you addressed at line 49.